app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -353,7 +353,7 @@
 
     <string name="status_wallet_connect_wrongversion">節點版本不相容 - 請盡速更新！</string>
 
-    <string name="menu_info">詳細資訊</string><!--Changed to: Show Secrets!-->
-    <string name="menu_streetmode">Street Mode</string>
-    <string name="info_streetmode_enabled">Street Mode enabled, tap for more info.</string>
+    <string name="menu_info">顯示私鑰資訊</string><!--Changed to: Show Secrets!-->
```

Comment:
Should we remove the comment after revising?

Reply:
I'm not sure either, maybe ping @m2049r

Reply:
I would remove it. I think he put it there to guide translators, but it lose its meaning once that string is correctly translated. Let's wait for him anyway

Reply:
yes, what @erciccione said :)